src/com/webonise/carbuilder/ColorFactory.java

```diff
@@ -0,0 +1,40 @@
+package com.webonise.carbuilder;
+
+public class ColorFactory extends AbstractCarFactory{
```

Comment:
i dont understand  the data modelling  here.You are equating a color to a car???